SECURITY.md

```diff
@@ -0,0 +1,39 @@
+# Security Notice
+
+This is the security notice for all DfE (Department for Education) repositories. The notice explains how vulnerabilities should be reported to the DfE. At the DfE there are cyber security and information assurance teams, as well as security-conscious people within the programmes, that assess and triage all reported vulnerabilities.
+
+## Reporting a Vulnerability
+
+The Department for Education is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it.
+
+Send details to [admin@digital.education.gov.uk](mailto:admin@digital.education.gov.uk), including:
```

Comment:
Pending progress with #3, perhaps a link to the generic gov.uk reporting page could be used in lieu of an email address?

```
https://www.gov.uk/help/report-vulnerability
```

Reply:
Yep - sounds like a good idea.

Hoping @lwlsns can assist with #3 😁